### PR TITLE
Feature/boilerplate

### PR DESCRIPTION
**Pull request by odeimaiz**
Linter added to boilerplate js-client (and fixed)

### CODE REVIEW

**Comment by sbenkler** [comment]
As already explained to Odei, I think the current approach via travis's matrix feature is not maintanable for the final repo.

 - too many different flavors of check would need to be added to travis
 - as an other developer not working in this project, it is difficult to build/run/test this project without digging deep into which packages need to be included in the host OS
 - for clang and cpp-linter it was even hard to sync the linter version from travis with the one in Ubuntu 16.04 ... that is why I finally gave up to do it in travis but creating a wrapper script dealing with a build-container.

It is up to you to accept this part. However, I would not go in that direction for the final repo due to maintenance issues.

**Comment by odeimaiz** [comment]
I find the current approach too complicated, that's why I added the matrix feature.
We could give it a try and see how it scales...

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "lintBoilerplate": "./node_modules/.bin/eslint -c ./templates/boilerplate/client-qx/.eslintrc.json ./templates/boilerplate/client-qx/source/class/qxapp/**/*.js"
+  }
```
Comment (oetiker) [inline package.json]: you are aware of the `qx lint` command ?

Comment (pcrespov) [reply package.json]: Is qx linter adding linting capabilities specific to qx on top of eslint or is a replacement?

**Pull request merged**